cmd/appliance/appliance.go

```diff
@@ -39,6 +39,7 @@ func NewApplianceCmd(f *factory.Factory) *cobra.Command {
 	cmd.AddCommand(files.NewFilesCmd(f))
 	cmd.AddCommand(maintenance.NewMaintenanceCmd(f))
 	cmd.AddCommand(NewSeedCmd(f))
+	cmd.AddCommand(NewForceDisableControllerCMD(f))
```

Comment:
```suggestion
	cmd.AddCommand(NewForceDisableControllerCmd(f))
``` 
minor nitpick, but for consistency with rest of the command funcs, I'd prefer `Cmd` over `CMD`